frontend/src/pages/privacy.tsx

```diff
@@ -1,235 +1,143 @@
+            information about some of our users. We are committed to protecting your personal information and your right to privacy. If you have any questions or concerns about
+            this privacy notice or our practices with regard to your personal information, please contact us at info@climatepolicyradar.org.
+          </p>
+          <p>
+            This privacy notice describes how we might use your information if you use our app website or use our data products at{" "}
+            <a href="https://climatepolicyradar.org/">https://climatepolicyradar.org/</a>
+          </p>
+          <h2>What Information do we collect?</h2>
+          <p>We collect personal information that you voluntarily provide through a number of entryways. These include:</p>
+          <ul>
+            <li>When you register on the Website</li>
+            <li>When you express an interest in obtaining information about us or our products and services</li>
+            <li>When you participate in activities on the Website</li>
+            <li>When you contact us</li>
+          </ul>
+          <p>
+            The personal information that we collect depends on the context of your interactions with us and the Website, the choices you make and the products and features you
+            use.
+          </p>
+          <p>The types of personal information that we process include:</p>
+          <ul>
+            <li>Your name</li>
+            <li>Postal address</li>
+            <li>Email address</li>
+            <li>Telephone numbers</li>
+            <li>Passwords</li>
+            <li>Contact preferences</li>
+          </ul>
+          <h2>Information automatically collected</h2>
+          <p>
+            We automatically collect certain information when you visit, use or navigate the Website. This information does not reveal your specific identity (like your name or
+            contact information) but may include device and usage information, such as your IP address, browser and device characteristics, operating system, language preferences,
+            referring URLs, device name, country, location, information about how and when you use our Website and other technical information. This information is primarily needed
+            to maintain the security and operation of our Website, and for our internal analytics and reporting purposes.
+          </p>
+          <p>
+            We also collect information through cookies and similar technologies. See our cookie policy <a href="/cookie-policy">here</a>.
+          </p>
+          <h2>How we use your information</h2>
+          <p>
+            Climate Policy Radar collects and processes your personal information, provided as detailed above, as necessary in order to provide you with our service and any related
+            services you may request, as well as to:
+          </p>
+          <ul>
+            <li>Facilitate account creation and login process</li>
```

Comment:
this may need to change

Reply:
Offline discussion to decide it is OK to remain for now as it covers the historic functionality.